transfermethodrepository/src/main/java/com/hyperwallet/android/ui/transfermethod/repository/TransferMethodRepositoryImpl.java

```diff
@@ -288,4 +295,12 @@ public Handler getHandler() {
             }
         });
     }
+
+    // Note: This way of surfacing error is not ideal but rather a workaround please have a look on other options,
+    // before resulting into this pattern
+    private HyperwalletErrors getErrorsOnUnsupportedTransferType() {
+        HyperwalletError error = new HyperwalletError(R.string.error_unsupported_transfer_type,
+                EC_UNEXPECTED_EXCEPTION);
+        return new HyperwalletErrors(Arrays.asList(error));
```

Comment:
Suggestion: we could simplify more.

Reply:
```
return new HyperwalletErrors(Arrays.asList(
        new HyperwalletError(R.string.error_unsupported_transfer_type, EC_UNEXPECTED_EXCEPTION)));
```
you mean that one? ☝️, looks unreadable to me, if we are going that path we can also do the following:
```
default: // error on unknown transfer type
      callback.onError(new HyperwalletErrors(Arrays.asList(
        new HyperwalletError(R.string.error_unsupported_transfer_type, EC_UNEXPECTED_EXCEPTION))));
```
but I rather make it cleaner and readable, easy to understand what's going on too, what do you think?

Reply:
okay @peter-joseph